python/ray/tune/trainable.py

```diff
@@ -306,32 +309,42 @@ def _train(self):
     def _save(self, checkpoint_dir):
         """Subclasses should override this to implement save().
 
+        See also: ray.tune.Trainable.save_dict.
```

Comment:
I'm not super sure that's useful -- how about we say you can also return a dict, and if so it will be auto saved?